pkgs/build-support/rust/default.nix

```diff
@@ -15,7 +15,13 @@
 }:
 
 { name ? "${args.pname}-${args.version}"
-, cargoSha256 ? "unset"
+
+  # SRI hash
+, cargoHash ? ""
```

Comment:
This flag needs to be documented in `doc/languages-frameworks/rust.section.md`

Reply:
I added a commit to add documentation for `cargoHash`.